src/containers/PastEvents.tsx

```diff
@@ -22,20 +29,23 @@ interface PastEventsContainerProps {
       start: string;
     },
   ];
+  fetchAttendance: Function;
   fetchPastEvents: Function;
 }
 
 const PastEventsContainer: React.FC<PastEventsContainerProps> = (props) => {
-  const { auth, events } = props;
+  const { auth, events, attendance } = props;
 
   useEffect(() => {
     props.fetchPastEvents();
+    props.fetchAttendance();
   }, []);
```

Comment:
Is there a reason why the functions here can be destructured and have to be called with the props?

Reply:
@leestanley There's an issue because the functions are imported at the top of the file, so if the variables were destructured, they would be seen as being redeclared even though the functions passed through props have been connected to redux.